### PR TITLE
Fix golint failures of test/e2e/network

### PR DESCRIPTION
**Pull request by SataQiu**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:
Fix golint failures of test/e2e/network

**Which issue(s) this PR fixes**:

Ref #68026

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/network/no_snat.go [hunk 1]
```diff
@@ -29,8 +29,8 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/kubernetes/test/e2e/framework"
 
-	. "github.com/onsi/ginkgo"
-	// . "github.com/onsi/gomega"
+	"github.com/onsi/ginkgo"
+	// "github.com/onsi/gomega"
```
Comment (spiffxp) [inline test/e2e/network/no_snat.go]: I would prefer we remove this instead of leaving it commented out

File: test/e2e/network/util_iperf.go [hunk 2]
```diff
@@ -101,7 +103,7 @@ func (s StrSlice) get(i int) string {
 func intOrFail(debugName string, rawValue string) int64 {
 	value, err := strconv.ParseInt(rawValue, 10, 64)
 	if err != nil {
-		framework.Failf("Failed parsing value %v from the string '%v' as an integer", debugName, rawValue)
+		framework.Failf("ginkgo.Failed parsing value %v from the string '%v' as an integer", debugName, rawValue)
```
Comment (spiffxp) [inline test/e2e/network/util_iperf.go]:
```suggestion
		framework.Failf("Failed parsing value %v from the string '%v' as an integer", debugName, rawValue)
```

**Review by spiffxp — CHANGES_REQUESTED** [review]
Two nits otherwise looks good

File: test/e2e/network/util_iperf.go (hunk 2 above)
Comment (SataQiu) [reply test/e2e/network/util_iperf.go]: Thanks a lot :+1: 
I've updated the code.

**Review by spiffxp — APPROVED** [review]

**Pull request merged**